build-system/babel-plugins/babel-plugin-transform-jss/create-hash.js

```diff
@@ -19,5 +19,14 @@ const crypto = require('crypto');
 // This is in its own file in order to make it easy to stub in tests.
 module.exports = {
   createHash: (filepath) =>
-    crypto.createHash('sha256').update(filepath).digest('hex').slice(0, 7),
+    crypto
+      .createHash('sha256')
+      .update(toPosix(filepath))
+      .digest('hex')
+      .slice(0, 7),
 };
+
+// To support Windows, use posix separators for all filepath hashes.
+function toPosix(filepath) {
+  return filepath.replace(/\\\\/g, '/').replace(/\\/g, '/');
```

Comment:
We can do this in a single replace with:
```suggestion
  return filepath.replace(/\\\\?/g, '/');
```

Reply:
Neat! I'm hoping to get rid of the fn entirely, but if not I'll update